redis/cluster.py

```diff
@@ -1078,11 +1061,9 @@ def _execute_command(self, target_node, *args, **kwargs):
                     )
                 return response
 
-            except (RedisClusterException, BusyLoadingError, AuthenticationError) as e:
-                log.exception(type(e))
-                raise
+            except AuthenticationError as e:
```

Comment:
Isn't catching & re-raising redundant?

Reply:
Okay so would you like me to remove those except blocks? I wasn't sure if you want to catch the specific `AuthenticationError`?